Add module arguments support for AutoLoad. Module arguments must be passed by separating with '/'.
Example:- $(call AutoLoad,30,testmod/debug=1/level=4)
Example with dependency:- $(call AutoLoad,40,xyzmod abcmod/debug=1 mymod/param1=2/param2=20)

diff --git a/include/kernel.mk b/include/kernel.mk
index 300da0d..c8cdead 100644
--- a/include/kernel.mk
+++ b/include/kernel.mk
@@ -107,7 +107,7 @@ define ModuleAutoLoad
 		shift 3; \
 		for mod in $$$$$$$$mods; do \
 			mkdir -p $(2)/etc/modules.d; \
-			echo "$$$$$$$$mod" >> $(2)/etc/modules.d/$$$$$$$$priority-$(1); \
+			echo "$$$$$$$$mod"|sed 's/\// /g' >> $(2)/etc/modules.d/$$$$$$$$priority-$(1); \
 		done; \
 		if [ -e $(2)/etc/modules.d/$$$$$$$$priority-$(1) ]; then \
 			if [ "$$$$$$$$boot" = "1" ]; then \
